packages/block-editor/src/components/inserter/style.scss

```diff
@@ -143,11 +143,13 @@ $block-inserter-search-height: 38px;
 	margin-right: 20px;
 	padding: 20px;
 	background: $white;
+	box-shadow: $shadow-popover;
+	border-radius: $radius-block-ui;
 
 	@include break-medium {
 		position: absolute;
-		top: 0;
-		left: calc(100% + 20px);
+		top: -1px;
```

Comment:
You could do `-$border-width` here instead of -1px, because the reason it's offset by one is because of the border, as best I can tell, that helps "document" the code.